manifest.toml

```diff
@@ -58,6 +58,12 @@ ram.runtime = "50M"
     type = "boolean"
     default = false
 
+    [install.codeerror]
+    ask.en = "Define a path about the 404 error page"
+    ask.fr = "Definir un chemin pour la page d'erreur 404"
+    type = "path"
+    default = "/404.html"
```

Comment:
Cheers !

I would tend to have this option only in the config panel, because this is an advanced option and otherwise going to confuse people

Reply:
True, set with a default value and only a setup available with the config panel 👍